problems/removeVowels.js

```diff
@@ -6,8 +6,10 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
-
+const removeVowels = (str) => {
+    let vowels = /[aeiou]+/gi
```

Comment:
This answer is very very slick, but make sure you understand 100% how this RegEx is working and could replicate this answer on an interview.